src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -291,6 +291,20 @@ type LogoutCommand = {
 
 // Build Image Command Types
 
+/**
+ * Target platform for the image build
+ */
+export type ContainerPlatform = undefined | {
```

Comment:
```suggestion
export type ContainerPlatform = {
```
Since the `platform` option is optional (has `?`), we don't need to define this type as potentially `undefined`

Reply:
this makes sense!